### PR TITLE
add troubleshot instructions when plugins.ts is missing

### PR DESCRIPTION
**Pull request by Sarabadu**
Signed-off-by: Juan Pablo Garcia Ripa <sarabadu@gmail.com>

## Hey, I just made a Pull Request!

Starting from zero and trying to add the shortcuts plugin was no clear because the create-app does not include a `plugins.ts` file as default now and is necessary for this plugin

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by timbonicus** [comment]
`plugins.ts` is not needed any more, actually! Sorry for the outdated instructions here. Since #3461 plugins are discovered through the React application tree, so simply adding the `<Shortcuts />` component should be enough.

File: plugins/shortcuts/README.md [hunk 1]
```diff
@@ -50,3 +54,33 @@ export const apis = [
   }),
 ];
 ```
+
+# Troubleshoot
```
Comment (timbonicus) [inline plugins/shortcuts/README.md:58]:
```suggestion
# Troubleshooting
```

File: plugins/shortcuts/README.md [hunk 2]
```diff
@@ -4,20 +4,24 @@ The shortcuts plugin allows a user to have easy access to pages within a Backsta
...
+
+This plugin requires explicit registration, so you will need to add it to your App's `plugins.ts` file:
 
 ```ts
 // ...
 export { shortcutsPlugin } from '@backstage/plugin-shortcuts';
 ```
 
-Add the `<Shortcuts />` component within your `<Sidebar>`:
+If you don't have a `plugins.ts` see: [troubleshoot](#troubleshoot)
```
Comment (timbonicus) [inline plugins/shortcuts/README.md:22]:
```suggestion
If you don't have a `plugins.ts` see: [troubleshooting](#troubleshooting)
```

File: plugins/shortcuts/README.md [hunk 3]
```diff
@@ -50,3 +54,33 @@ export const apis = [
   }),
 ];
 ```
+
+# Troubleshoot
+
+If you don't have a `plugins.ts` you can create a new one on `packages/app/src/plugins.ts` and then add this lines to your `App.tsx` if there is other plugins that requires explicit registration:
```
Comment (timbonicus) [inline plugins/shortcuts/README.md:60]:
```suggestion
If you don't have a `plugins.ts` file, you can create it with the path `packages/app/src/plugins.ts` and then import it into your `App.tsx`:
```

**Review by timbonicus — COMMENTED** [review]
Some minor language suggestions -- looks good otherwise!

**Review by timbonicus — APPROVED** [review]
Lovely!

**Comment by jhaals** [comment]
Thanks @Sarabadu and @timbonicus for a splendid review.

**Pull request merged**